vibi-dpu/src/core/coverage.rs

```diff
@@ -25,20 +25,46 @@ pub async fn process_coverage(hunkmap: &HunkMap, review: &Review, repo_config: &
                 // create comment text
                 let comment = comment_text(coverage_map, repo_config.auto_assign());
                 // add comment
-                add_comment(&comment, review, &access_token).await; 
+                add_comment(&comment, review, &access_token).await;
             }
             if repo_config.auto_assign() {
-                // add reviewers
-                println!("Adding reviewers...");
                 let mut author_set: HashSet<String> = HashSet::new();
                 author_set.insert(prhunk.author().to_string());
                 for blame in prhunk.blamevec() {
-                    if author_set.contains(blame.author()) {
+                    let blame_author_opt = author_from_commit(blame.commit(),
+                        hunkmap.repo_name(), hunkmap.repo_owner()).await;
+                    if blame_author_opt.is_none() {
+                        eprintln!("[process_coverage] Unable to get blame author for commit: {}", &blame.commit());
                         continue;
                     }
-                    author_set.insert(blame.author().to_string());
-                    let author_id = blame.author();
-                    add_reviewers(blame.author(), review, &access_token).await;
+                    let blame_author = blame_author_opt.expect("Empty blame_author_opt");
+                    let user_key = blame_author.display_name().to_string();
+                    let blame_user_opt = get_workspace_user_from_db(&user_key);
```

Comment:
Please don't call this `blame_user_opt` - it has nothing to do with the blame or the diff.

I suggest `bb_workspace_user`